test/python/picongpu/compiling/simulation.py

```diff
@@ -56,6 +67,12 @@ def test_minimal(self):
         runner.build()
         runner.run()
 
+    def test_moving_window(self):
+        picmi_sim = self._set_up_sim(picongpu_moving_window_move_point=0.9)
+        runner = pypicongpu.Runner(picmi_sim)
+        runner.generate(printDirToConsole=True)
+        runner.build()
+
```

Comment:
Do we also need a runner.run() here?

Reply:
yes, you are right. Since Moving Window configuration is run time only we need to do a short run to test

Reply:
fixed

Reply:
But we can't since our CI jobs only ever have access to one device therefore the PyPIConGPU runner can't run a simulation with moving window, since we require at least 2 devices for a moving window.

We could use the `-r 2` option to run a moving window simulation but the PyPIConGPU runner can't do that currently.

Therefore would shift the run time tests to a future PR.

Reply:
Sounds okay. Could you please create an issue to track the addition of the runtime test and I will merge this PR as is.

Reply:
created the Issue #4995 to track the missing feature.